.github/auto_assign.yml

```diff
@@ -0,0 +1,31 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: false
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 1
+
+# A number of assignees to add to the pull request
+# Set to 0 to add all of the assignees.
+# Uses numberOfReviewers if unset.
+# numberOfAssignees: 2
+
+# Set to true to add reviewers from different groups to pull requests
+useReviewGroups: true
+
+# A list of reviewers, split into different groups, to be added to pull requests (GitHub user name)
+reviewGroups:
+  projectAdmin:
+    - InbasasiS
+    - AkbarHabeeb
+    - Inbasasi
+  groupB:
```

Comment:
"groupB" should be name as mentors.

Reply:
I'll do that once I'll get mentor's details